Footer/Footer 11 lift blue.html

```diff
@@ -0,0 +1,95 @@
+<footer class="bg-blue-600 antialiased">
+  <div class="container mx-auto pt-20 pb-14">
+    <div class="max-w-2xl mx-auto">
+      <h1 class="text-center text-4xl font-bold mb-6 text-white">
+        Seasoned. Nimble. Remote.
+      </h1>
+      <p class="text-center font-normal text-sm leading-6 text-white/80 mb-8">
+        We’re a diverse and passionate team that takes ownership of your design
+        and empower you to execute the roadmap. We stay light on our feet and
+        truly enjoy delivering great work.
+      </p>
+      <div class="w-100 flex justify-center gap-x-8">
+        <a
+          class="inline-flex items-center bg-gray-100 py-1.5 px-4 rounded-full"
+          href=""
```

Comment:
use `href="#"` for empty link redirections

Reply:
Thank you for your suggestion. I will commit these changes ASAP